public/css/addVillager.css

```
@@ -0,0 +1,187 @@
+@import "reset.css";
+
+body {
+  background-image: url("../assets/graphics/addVillager-backgroundImg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.header-logo {
+  width: 100%;
+  max-width: 20rem;
+  height: auto;
+  margin-top: 2rem;
+}
+
+.profile-wrap {
+  width: 70%;
+  height: 44rem;
+  background-color: rgba(252, 241, 224, 0.8);
+  margin: 1rem auto;
+  border-radius: 3rem;
+  border: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.header-text {
+  color: #3f5d4d;
+  margin-top: 3rem;
+}
+
+.profile-main {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  padding: 1rem 0;
+  align-items: flex-start;
```

저번에 리뷰 안 남겼었나요?!

```suggestion
  display: flex;
  align-items: flex-start;
  flex-direction: row;
  
  margin: 0 auto;
  padding: 1rem 0;
  
  width: 80%;
```

이렇게 개행을 추가하면 가독성이 좋아집니다. 

그리고 align-items와 flex-direction은 불필요해보이네요

---

다른분의 리뷰에 다셨던 것 같은데 제가 다른분 리뷰를 제대로 확인하지 못한 것 같아요!..😢
개행 추가한 버전으로 수정했습니다!

https://github.com/KDT1-FE/Y_FE_JAVASCRIPT_PICTURE/pull/12/commits/4bd2c5d202cf313aa6737daca28610662150b342